### ISSUE TITLE
Transcript doesn't show in 7.1.8.

### ISSUE THREAD

**Issue by brickviking** [dolphinsmalltalk/Dolphin#802]
**Describe the bug**
Clicking on System Transcript in either the System Folder or from the toolbar or menu doesn't result in a Transcript window.

**To Reproduce**
1.Click Tools menu
2. Click on 'System Transcript'

**Expected behavior**
A Transcript window should appear.

**Please complete the following information):**
 - OS: 32-bit Vista, 64-bit Fedora 28.

**Additional context**
Latest 7.1.8 VM, commit 53eb8cc35b6588d3aa7479be959917e7defd1fc4.

**Comment by brickviking** [comment]
I stepped through Transcript class>>current with 7.1.3 and 7.1.8, and I got to this stage in the process. Sorry about the image quality, but it's the best I know how to do currently.

___Showing debug for  7.1.3 and 7.1.8 until it differs. Here's where it differs___

![Transcript-7 1 3_and_7 1 8](https://user-images.githubusercontent.com/62885/62843288-e920aa00-bd0c-11e9-8505-2da7354a893c.png)


I'm not sure why the DNUs are being handled this way, given that they're deliberate instead of uhm, accidental. I'll chalk it up to a lack of ST experience.

**Comment by blairmcg** [comment]
Doesn't repro for me; the Transcript opens as expected. If you are working with a freshly booted image and a freshly built VM (or fetchvm), there is no reason it wouldn't work.

To repair the image you have evaluate:
```
    Transcript := DeafObject current.
```

**Comment by brickviking** [comment]
Hm, seems weird, but that works. Okay, I'll try again on an absolutely fresh image, though I wish I knew how I torpedoed the Transcript in the image I currently have. Thanks for that, I'll call this closed.

**Comment by brickviking** [comment]
I've just managed to trigger the "bug" again....and I think I found what the trigger is.

Start with fresh DPRO, hit Transcript. It works fine. So, hit the Panic, click "Yes", then a new System Folder window will open. All good so far. Now try bringing up a Transcript. Nothing appears, until I evaluate "Transcript := DeafObject current" in a Workspace.

So far, I've been able to reproduce this in 7.1.4 (Linux) and 7.1.8 (Linux). I'm about to go try them both under Vista. For comparison, 7.1.3 and 7.0.57.2 didn't show the same behaviour.
